### PR TITLE
Adding recipe for fajitas

### PR DESCRIPTION
**Pull request by brianamarie**
Hello @githubteacher @beardofedu! This pull request adds a recipe for fajitas. 

What do you think?

### CODE REVIEW

**Comment by githubteacher** [comment]
@brianamarie Hey, can you please resubmit to the `development` branch?

However, for the sake of time, I don't need you to!

File: 02-02-2017-fajitas.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+- Ingredient
+- Ingredient
+- 1 large onion 
+- 1 medium red bell pepper
+- 1 lb boneless skinless chicken breasts 
+- 1 package (1 oz) Old El Paso™ fajita seasoning mix
+- 2 tablespoons vegetable oil
+- Old El Paso™ tortillas for soft tacos & fajitas (6 inch; from 8.2-oz package)
+- Sour Cream
+- Old El Paso™ Thick 'n Chunky salsa -Chopped fresh cilantro
```
Comment (githubteacher) [inline 02-02-2017-fajitas.md]: Please move `Chopped fresh cilantro` to another line.

File: 02-02-2017-fajitas.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+## Directions
+
+1. Heat oven to 400°F. Spray 13x9-inch (3-quart) glass baking dish with cooking spray. Cut onion and bell pepper into even slices; place in baking dish.
+1. Cut chicken breasts into thin strips; add to vegetables in dish. Sprinkle with seasoning mix; drizzle with oil. Stir until combined and pieces are coated.
+1. Bake 35 to 40 minutes, stirring once halfway through baking, until chicken is no longer pink in center.
+4 Spoon small amount of chicken and veggies onto each tortilla. Top each with sour cream, salsa and cilantro, or your favorite fajita toppings.
```
Comment (githubteacher) [inline 02-02-2017-fajitas.md]: Please make `Step 4` an ordered list item.

**Review by githubteacher — CHANGES_REQUESTED** [review]
@brianamarie once you fix these minor issues, it will be good to 🚢

**Comment by brianamarie** [comment]
@githubteacher I've made some edits addressing your comments. Does it look better now?

**Review by teresa1688 — APPROVED** [review]
looks good

**Review by githubteacher — APPROVED** [review]

**Comment by githubteacher** [comment]
@brianamarie good to 🚢 click the big green!

**Review by teresa1688 — COMMENTED** [review]
good

**Pull request merged**